lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,410 @@
+
+  // Current status of the buffer.
+  var _status = Status.addPoints;
+
+  // Buffer radius.
+  var _bufferRadius = 100.0;
+
+  // Union status.
+  var _shouldUnion = false;
+
+  // A flag for when the settings bottom sheet is visible.
+  var _showSettings = false;
+
+  // Define the graphics overlays.
+  final bufferGraphicsOverlay = GraphicsOverlay();
+  final tapPointGraphicsOverlay = GraphicsOverlay();
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  final statePlanNorthCentralTexasSpatialReference =
+      SpatialReference(wkid: 32038);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                // Display a banner with instructions at the top.
+                SafeArea(
+                  child: IgnorePointer(
+                    child: Container(
+                      padding: const EdgeInsets.all(10),
+                      color: Colors.white.withOpacity(0.7),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Text(
+                            _status.label,
+                            textAlign: TextAlign.center,
+                            style: const TextStyle(color: Colors.black),
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                ),
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        setState(() => _showSettings = !_showSettings);
+                      },
+                      child: const Text('Settings'),
+                    ),
+                    // A button to clear the buffers.
+                    ElevatedButton(
+                      onPressed: _bufferPoints.isEmpty
+                          ? null
+                          : () {
+                              clearBufferPoints();
+                              setState(() => _status = Status.addPoints);
+                            },
+                      child: const Text('Clear'),
+                    ),
+                  ],
+                ),
+                // Conditionally display the settings.
+                if (_showSettings) buildSettings(context, setState),
```

Comment:
In other samples, we take advantage of the "bottomSheet" parameter on Scaffold:
```dart
      bottomSheet: _settingsVisible ? buildSettings(context) : null,
```
I like the "setState" thing here, but otherwise should we use "bottomSheet:"?

Reply:
changed it to `bottomSheet:`. It's working as expected.